CrossPlatform.sln

```diff
@@ -1,358 +0,0 @@
-
```

Comment:
@agocke If we remove CrossPlatform.sln, I assume this means `dotnet build Roslyn.sln` and `dotnet build Compilers.sln` work on Linux?